POCServer/src/java/Resources/Model/Rol.java

```diff
@@ -0,0 +1,52 @@
+package Resources.Model;
+
+import Model.Roles;
+import Model.UserRoles;
+
+public class Rol {    
+    private int id;
+    private String name;
+    private String description;
+
+    public Rol(UserRoles userRoles) {
+        this.id = userRoles.getRoles().getId();
+        this.name = userRoles.getRoles().getName();
+        this.description = userRoles.getRoles().getDescription();
+    }
+    
+    public Rol(Roles roles) {
+        this.id = roles.getId();
+        this.name = roles.getName();
+        this.description = roles.getDescription();
+    }
+
+    public Rol(int id, String name, String description) {
+        this.id = id;
+        this.name = name;
+        this.description = description;
+    }        
+    
+    public int getId() {
```

Comment:
Only as comment here you can use Lombok to avoid create setters and getters for each property in the class. Please review documentation about Lombok, because we use it in the backend project.